pro_wes/ga4gh/wes/workflow_runs.py

```diff
@@ -424,7 +427,7 @@ def _create_run_environment(
             # populate document
             document.run_log.run_id = run_id
             document.task_id = uuid()
-            document.work_dir = work_dir.as_posix()
+            document.work_dir = str(work_dir)  # type: ignore
```

Comment:
Sorry, but I'm a bit confused. What's the expected type for `document.work_dir`, and why don't you assign it the type it wants?